packages/tooling/fast-tooling/src/message-system-service/message-system.service.ts

```diff
@@ -81,14 +94,14 @@ export abstract class MessageSystemService<C> {
      * The service should get any config to be passed to
      * the registered actions
      */
-    abstract getActionConfig(id: string): C;
+    abstract getActionConfig(id: string): A;
```

Comment:
I know this is technically just a typing change, but curious if this is backwards compatible or breaking (are A and C "ducks")?

Reply:
*Not exported - disregard (thx @Janechu)